it/src/main/java/com/google/cloud/teleport/it/dataflow/AbstractDataflowTemplateClient.java

```diff
@@ -55,6 +55,14 @@ public void cancelJob(String project, String region, String jobId) throws IOExce
     client.projects().locations().jobs().update(project, region, jobId, job).execute();
   }
 
+  @Override
+  public void drainlJob(String project, String region, String jobId) throws IOException {
```

Comment:
done.

We need to be able to drain the job because for pipeline updates in syndeo, we are going to drain-then-rerun, so we need to verify proper job semantics.